_sass/bootstrap/_variables.scss

```diff
@@ -212,7 +212,7 @@ $container-max-widths: (
   sm: 540px,
   md: 720px,
   lg: 960px,
-  xl: 1360px
+  xl: 1140px
```

Comment:
Given the smaller max-width for containers we should update the widths of the event cards. I had tweaked them to look OK-ish, now they're too small: https://deploy-preview-230--antennapod-test.netlify.app/events/

Similarly, we might want to limit the number of ['Contribute' cards](https://deploy-preview-230--antennapod-test.netlify.app/contribute/) to 3 (rather than 4).

Reply:
It seems that bootstrap 4.5.2 was modified. I restored the vanilla bootstrap 4.5.2 and moved the customization to `_variables.scss`.
Now the result looks like before, except the corner radius.

Reply:
Actually I prefer the vanilla bootstrap - it's less wide which makes it easier to read. So I would keep that and adjust the number of event and contribute cards accordingly.

Reply:
I would favor leaving this PR as a mostly technical upgrade and consider the visual changes when migrating to bootstrap v5.